frontend/src/components/SectionCard.tsx

```diff
@@ -85,11 +103,7 @@ const SectionCard = (props: ISection) => {
           </Stack>
           <Stack direction="row">
             <Typography>
-              {props.suggestions.map((suggestion: ISuggestion) => {
-                if (suggestion.content) {
-                  return <ContentEditor initialContent={suggestion.content} readonly={true} />;
-                }
-              })}
+              <div dangerouslySetInnerHTML={{ __html: currentSuggestionHTML }} />
```

Comment:
Did the previous implementation fail to render the content?

Reply:
The editor in read-only mode makes an ugly flicker in the UI when switching the card between suggestions, because it isn't a true React component and it takes a moment to load its contents. I assume this is one of the reasons why Tiptap provided this HTML rendering package.

If we don't mind the flicker then you can simply revert back to commit [46b5d2b](https://github.com/consenz-io/webapp/pull/143/commits/46b5d2b4a6b84952d42ca561f0a0797e6e249abc) and the version with the read-only editor there also works!